api/working-with-extensions/continuous-integration.md

```diff
@@ -157,7 +157,7 @@ jobs:
 
 ### GitHub Actions automated publishing
 
-1. Set up `VSCE_PAT` as an encrypted secret using the [GitHub Actions secrets instructions](https://help.github.com/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository).
+1. Set up `VSCE_PAT` as an encrypted secret using the [GitHub Actions secrets instructions](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository)).
```

Comment:
Thanks for updating this link.
Can you remove `en/` from the URL so that readers will get the appropriate localized version of the topic, if available?

Reply:
Done, thanks!

Reply:
Oops, I missed the extra parentheses during my initial review. Can you fix that up as well? Thanks

![image](https://user-images.githubusercontent.com/12818376/231001283-073d1983-9b93-484e-967c-0f6513637afc.png)

Reply:
I've merged the PR and I'll remove the extra parentheses in a follow up PR.
Thanks again for noticing and updating the link!